package.json

```diff
@@ -33,6 +33,8 @@
     "@kiwicom/orbit-components": "^0.23.0",
     "@kiwicom/orbit-design-tokens": "^0.2.5",
     "react-native-shimmer": "^0.4.1",
+    "intl": "^1.2.5",
```

Comment:
This should be removed, right? @RobinCsl

Reply:
That's right, a component library should not be concerned with l10n and i18n apart from RTL support, in my opinion.

Reply:
@lukewalczak Could you please remove `intl` library?

Reply:
I thought I removed it :( removing right now